samples/create_table.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+from sqlalchemy import (
+        Column,
+        Integer,
+        MetaData,
+        String,
+        Table,
+        create_engine,
+)
+
+
+def create_table(url, random_table_id):
+    """Create the table"""
```

Comment:
Please ensure all docstrings end with a full stop:
```suggestion
    """Create the table."""
```